app/views/instructeurs/dossiers/_header_top.html.haml

```diff
@@ -26,3 +25,22 @@
   - if dossier.user_deleted?
     %p.fr-mb-1w
       %small L’usager a supprimé son compte. Vous pouvez archiver puis supprimer le dossier.
+
+  .fr-mb-3w
+    - if dossier.procedure_labels.present?
+      - dossier.procedure_labels.each do |label|
+        = tag_label(label.name, label.color)
+
+    = render Dropdown::MenuComponent.new(wrapper: :span, button_options: { class: ['fr-btn--sm fr-btn--tertiary-no-outline fr-pl-1v']}, menu_options: { class: ['dropdown-label left-aligned'] }) do |menu|
+      - if dossier.procedure_labels.empty?
+        - menu.with_button_inner_html do
+          Ajouter un label
+
+      - menu.with_form do
+        = form_with(url: dossier_labels_instructeur_dossier_path(dossier_id: dossier.id, procedure_id: dossier.procedure.id), method: :post, class: 'fr-p-3w', data: { controller: 'autosubmit', turbo: 'true' }) do |f|
```

Comment:
Je sais plus comment étaient les maquettes, mais je trouve que ce menu manque d'un titre (discret) en particulier lorsqu'on a déjà ajouté un premier label et qu'on ne voit plus le bouton "Ajouter un label". Sur github je vois _Apply labels to this pull request_ . Çæ pourrait être _Modifier les labels de ce dossier_